mne/io/eeglab/eeglab.py

```diff
@@ -445,3 +471,45 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
             reject=reject, flat=flat, reject_tmin=reject_tmin,
             reject_tmax=reject_tmax, add_eeg_ref=False, verbose=verbose)
         logger.info('Ready.')
+
+
+def _events_from_eeglab_raw(fname, event_id=dict()):
```

Comment:
you already have this: https://github.com/mne-tools/mne-python/blob/master/mne/io/eeglab/eeglab.py#L404